### DISCUSSION TITLE
Add Mastodon profile to Community settings (and, thus, to Header & Footer social media sections)

### DISCUSSION THREAD

**Discussion by tosteiner** [knowledgefutures/pubpub#2513] [Ideas]
With the increasing popularity of Mastodon as the major Fediverse alternative to Twitter, I was wondering if it would be possible to have a dedicated Mastodon line in the community settings, which would then auto-populate to Header and Footer areas in a similar way to what we currently have for Twitter?

In addition to that, if this could be rendered with the HTML `rel`-tag intact, this would help with direct integration with Mastodon's profile [link verification](https://docs.joinmastodon.org/user/profile/#verification) feature.

Screenshot of the area I'm referring to [here](https://cloud.flavoursofopen.science/index.php/s/MQDQWjgC5XNnecM).

**Comment by gabestein** [comment]
100%. Moving this to an issue and tagging it with quick wins so we can grab it opportunistically. I wanted this for gabestein.com (and rather hackily added the link in an HTML block at the bottom), and we've also had instagram requests.

**Reply by tosteiner** [reply]
wonderful, thanks, Gabe!

**Comment by timelf** [comment]
Hi Gabe, I just wanted to check in on links to socials. Since Toby suggested the Mastodon link last year, the social media landscape has further fragmented and proliferated, so it would have hard to come up with a stable list of platforms to build for. Is there a good strategy for incorporating links to these?

I didn't see this get into the roadmap for last year. What are the prospects for this year?
